onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -946,12 +946,25 @@ def aten_batch_norm_update_stats(
     raise NotImplementedError()
 
 
-def aten_bernoulli(self: TensorType, generator: Optional[str] = None) -> TensorType:
+def aten_bernoulli(self: TensorType) -> TensorType:
     """bernoulli(Tensor self, *, Generator? generator=None) -> Tensor"""
 
     raise NotImplementedError()
 
 
+@torch_op("aten::bernoulli.p")
+def aten_bernoulli_p(self: TensorType, p: float) -> TensorType:
+    """aten::bernoulli.p(Tensor self, float p, *, Generator? generator=None) -> Tensor"""
+    rands = op.RandomUniformLike(
+        self,
+        high=1.0,
+        low=0.0,
+    )
```

Comment:
Should we set `dtype=FLOAT.dtype`?

Reply:
Otherwise the Less below may feel a little flaky, when the input is float64 (not sure if there will be a castlike) or ints

Reply:
it's ok, `RandomUniformLike` always outputs floating types

Reply:
But it’s still going to output one of float16, 32 or 64. I wonder if we hit issues with Less (type mismatch). So far the test looks good though so I guess that’s ok.

Reply:
it seems we are saved by static_autocast since `p` is a float/ attribute so castlike is inserted .